interface/index.php

```diff
@@ -10,6 +10,7 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+$sessionAllowWrite = true;
```

Comment:
TODO
Will try removing this

Reply:
Gonna actually remove this file completely since on testing see no use for it in OpenEMR.